docs/user_guides/fs/feature_group/notification.md

```diff
@@ -79,7 +79,8 @@ Here is an example output:
 
 ```
 {
-  "projectId":119,  # project used for data ingestion
+  "projectName":"project_name",  # name of the project used for data ingestion
```

Comment:
I would change these to say that this is the project the feature group belongs to.

Reply:
Same for the project id